Lamina/include/GL/opengl.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef LM_OPENGL
+#define LM_OPENGL
+
+#include <glad/glad.h>
+#include <GLFW/glfw3.h>
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <sstream>
```

Comment:
Most of these includes are not necessary in this file and will increase build times. If you need them in the source file, you should include them there.